### DISCUSSION TITLE
Job postings for Dart

### DISCUSSION THREAD

**Discussion by parlough** [parlough/thisweekindart#4] [Ideas]
Are you hiring someone to write Dart code or have you seen a job posting requiring Dart experience? I understand if you don't want to share if you're applying yourself, but if your heart is feeling kind or you don't need the position, share it with the world!

We'd love to include reputable job postings in weekly issues as there are many Dartisans out there still flying around looking for the perfect position. Don't let their wings get tired and share the job information with us and we'll share with them. Thanks for helping out :>

**Requirements:**
- Job will actually utilize Dart in some fashion
- Job has a valid and still active job posting

**Include:**
- The position title
- The company
- The location(or lack thereof)
- The job posting link

**Comment by robrbecker** [comment]
Workiva has been building our web based product suite and platform using Dart since 2015. If you're a strong web developer and are into Dart, you should apply here https://workiva.wd1.myworkdayjobs.com/en-US/careers/details/Senior-Software-Engineer---Frontend-Frameworks_R5084?jobFamilyGroup=75715069c9720196b19bd65ab32bc01a
or to other listings you find interesting https://www.workiva.com/careers. Our entire frontend is written with Dart so any team you join will be using Dart.

We're remote friendly and have been in the top 100 Great Places To Work for the last 3 years. Read more about our culture https://www.greatplacetowork.com/certified-company/1383563

ping @parlough

**Reply by parlough** [reply]
Thanks @robrbecker, it's always exciting to see Workiva's investment around Dart. I'll be sure sure to include this listing in this week's issue!
